### PR TITLE
Fix doctests after QGIS commit 3d31477750

### PR DESCRIPTION
**Pull request by elpaso**
The renderer is now picky about mapsettings being
valid, since the test layer had only a single point
feature extent was empty and map settings were not
valid.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -395,7 +408,10 @@ def start_qgis():
     f = QgsFeature(iface.activeLayer().fields())
     f.setAttributes([1, 'First feature'])
     f.setGeometry(QgsGeometry.fromWkt('point( 7 45)'))
-    iface.activeLayer().dataProvider().addFeatures([f])
+    f2 = QgsFeature(iface.activeLayer().fields())
+    f2.setAttributes([1, 'First feature'])
```
Comment (DelazJ) [inline conf.py:412]: It's intentional that the two features have the same attributes?

**Review by DelazJ — COMMENTED** [review]
Thanks for the fix @elpaso

File: conf.py (hunk 1 above)
Comment (elpaso) [reply conf.py:412]: no, but it's not really relevant. I can change that though.

Comment (elpaso) [inline conf.py:412]:
```suggestion
    f2.setAttributes([2, 'Second feature'])
```

**Pull request merged**